src/Microsoft.Diagnostics.Monitoring/Configuration/LoggingSourceConfiguration.cs

```diff
@@ -18,7 +26,12 @@ public override IList<EventPipeProvider> GetProviders()
                 new EventPipeProvider(
                     MicrosoftExtensionsLoggingProviderName,
                     EventLevel.LogAlways,
-                    (long)(LoggingEventSource.Keywords.JsonMessage | LoggingEventSource.Keywords.FormattedMessage)
+                    (long)(LoggingEventSource.Keywords.JsonMessage | LoggingEventSource.Keywords.FormattedMessage),
+                    arguments: new Dictionary<string,string>
+                        {
+                            // Filter all loggers to the specified level
+                            { "FilterSpecs", $"*:{_level:G}" }
```

Comment:
So will this do _level and higher? or just _level? I'm curious how much overhead the filtering adds.

Reply:
This is reporting the specified level and higher. So by default, you get everything from Debug to Critical. If you specify Warning, then you get Warning to Critical.